app/presenters/vendor_api/application_presenter.rb

```diff
@@ -96,15 +96,16 @@ def references
     end
 
     def show_references?
-      version_1_3_or_above? || application_is_in_an_accepted_state?
+      (version_1_3_or_above? && !application_unsuccessful?) ||
```

Comment:
don't like asking whether an application is "not unsuccessful". is it reasonable to ask `application_successful?` instead, or you think misleading?

Reply:
I considered that but no, because there are many states which are neither successful nor unsuccessful.  In this case we're specifically hiding states considered "unsuccessful".

Reply:
fair